cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -388,6 +388,8 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
 
+/// Creates an AgentId from a MultiLocation. An AgentId is a unique mapping to a Agent contract
+/// which acts as the sovereign account for the MultiLocation.
```

Comment:
```suggestion
/// Creates an AgentId from a MultiLocation. An AgentId is a unique mapping to a Agent contract on Ethereum
/// which acts as the sovereign account for the MultiLocation.
```

Reply:
Resolved in https://github.com/Snowfork/polkadot-sdk/pull/81/commits/0d33404e71db15120519566dfade77dd36bd477c